packages/framework/ai-collab/src/aiCollabApi.ts

```diff
@@ -168,6 +170,11 @@ export interface AiCollabSuccessResponse {
 	 * {@inheritDoc TokenUsage}
 	 */
 	readonly tokensUsed: TokenUsage;
+	/**
+	 * A list of diffs that represent the changes made by the AI collaboration.
+	 * @remarks This is intended to be used to update your applications UI view with the changes made by the AI collaboration.
+	 */
+	readonly uiDiffs: Diff[];
```

Comment:
These diffs aren't really UI-specific. We can probably just call them "diffs" or "editDiffs"?

Reply:
Remove all Ui in names